.github/workflows/test.yml

```diff
@@ -8,10 +8,10 @@ jobs:
   tests:
     strategy:
       matrix:
-        os: [ubuntu-18.04, macos-10.15]
+        os: [ubuntu-18.04, ubuntu-20.04, macos-10.15]
```

Comment:
shall we remove `ubuntu-18.04`?

Reply:
I just removed 20.04 from this PR. Will make it into another.